modules/i18n/src/translations/si-LK/portals/console.ts

```diff
@@ -7488,60 +7491,60 @@ export const console: ConsoleNS = {
-                            message: "සංවිධානයට විද්යුත් තැපැල් වසම් සාර්ථකව එක් කළේය."
+                            description: "විද්‍යුත් තැපැල් වසම් සාර්ථකව එකතු කරන ලදී.",
+                            message: "සාර්ථකව එකතු කරන ලදි"
                         }
                     },
                     disableEmailDomainDiscovery: {
                         error: {
-                            description: "විද්‍යුත් තැපැල් වසම් සොයාගැනීම අක්‍රිය කිරීමේදී දෝෂයක් ඇති විය",
-                            message: "මොකක්හරි වැරැද්දක් වෙලා"
+                            description: "විද්‍යුත් තැපැල් වසම් සොයාගැනීම අක්‍රිය කිරීමේදී දෝෂයක් ඇති විය.",
+                            message: "අබල කිරීම අසාර්ථකයි"
                         },
                         success: {
-                            description: "විද්‍යුත් තැපැල් වසම් සොයා ගැනීම සාර්ථකව අක්‍රීය කරන ලදී",
-                            message: "විද්‍යුත් තැපෑල වසම් සොයා ගැනීම සාර්ථකව අක්‍රීය කර ඇත"
+                            description: "විද්‍යුත් තැපැල් වසම් සොයා ගැනීම සාර්ථකව අක්‍රීය කරන ලදී.",
+                            message: "සාර්ථකව අක්‍රියයි"
                         }
                     },
                     enableEmailDomainDiscovery: {
                         error: {
-                            description: "විද්‍යුත් තැපෑල වසම් සොයාගැනීම සබල කිරීමේදී දෝෂයක් ඇති විය",
-                            message: "මොකක්හරි වැරැද්දක් වෙලා"
+                            description: "විද්‍යුත් තැපෑල වසම් සොයාගැනීම සබල කිරීමේදී දෝෂයක් ඇති විය.",
+                            message: "සබල කිරීම අසාර්ථකයි"
                         },
                         success: {
-                            description: "විද්‍යුත් තැපැල් වසම් සොයාගැනීම සාර්ථකව සක්‍රීය කර ඇත",
-                            message: "විද්‍යුත් තැපැල් වසම් සොයා ගැනීම සාර්ථකව සක්‍රීය කර ඇත"
+                            description: "විද්‍යුත් තැපැල් වසම් සොයාගැනීම සාර්ථකව සක්‍රීය කර ඇත.",
+                            message: "සාර්ථකව සබල කර ඇත"
                         }
                     },
                     fetchOrganizationDiscoveryAttributes: {
                         error: {
-                            description: "සංවිධානයේ සොයාගැනීම් ගුණාංග ලබා ගැනීමේදී දෝෂයක් ඇති විය",
-                            message: "මොකක්හරි වැරැද්දක් වෙලා"
+                            description: "සංවිධානයේ සොයාගැනීම් ගුණාංග ලබා ගැනීමේදී දෝෂයක් ඇති විය.",
+                            message: "ලබා ගැනීම අසාර්ථකයි"
                         }
                     },
                     getEmailDomainDiscovery: {
                         error: {
-                            description: "ඊමේල් වසම් සොයාගැනීමේ වින්‍යාසය ලබා ගැනීමේදී දෝෂයක් ඇති විය",
-                            message: "මොකක්හරි වැරැද්දක් වෙලා"
+                            description: "ඊමේල් වසම් සොයාගැනීමේ වින්‍යාසය ලබා ගැනීමේදී දෝෂයක් ඇති විය.",
+                            message: "Retrieval unsuccessful"
```

Comment:
why this in english?

Reply:
Fixed with 45fedafa3c5b995e4def060f6399037dd4c4d9fd